interaction16/ViewModels/IXDASessionsViewModel.m

```diff
@@ -114,6 +184,30 @@ - (IXDASessionDetailsViewModel *)sessionsDetailViewModelOfArray:(NSArray *)selec
     return viewModel;
 }
 
+- (IXDASessionDetailsViewModel *)sessionsDetailViewModelWithEventKey:(NSString *)eventKey {
+    __block Session *session = nil;
```

Comment:
The code is good, alternatives could be:
- As you are not using the idx here, you could use simple for(Session *oneSession in self Sessions)
- You could also make it reactive, if you would like to avoid the Session *Session = nil with
   Session *Session = [self.session rac_sequence] filter or sth.
But this is really only gusto, so no need to change...